src/WindowsIotServices/WindowsIotServices.Autorest/README.md

```diff
@@ -85,14 +81,12 @@ directive:
     remove: true
   # Remove parameter set name
   - where:
-      verb: New
       subject: Service$
-      variant: Create$|CreateViaIdentity$|CreateViaIdentityExpanded$
+      variant: ^(Create|Update)(?!.*?(Expanded|JsonFilePath|JsonString))
     remove: true
   - where:
-      verb: Update
       subject: Service$
-      variant: Update$|UpdateViaIdentity$
+      variant: ^CreateViaIdentity.*$
```

Comment:
Please follow the screenshot below to update the directive.
![image](https://github.com/user-attachments/assets/e475b4d7-1ef1-42dd-af28-6dbbb281935d)